docs/api/web-contents.md

```diff
@@ -516,6 +516,15 @@ Emitted when the `WebContents` loses focus.
 
 Emitted when the `WebContents` gains focus.
 
+Note that on macOS, having focus means the `WebContents` is the first responder
+of window, so switching focus between windows would not trigger the `focus` and
+`blur` events of `WebContents`, as the first responder of each window is not
+changed.
```

Comment:
For a little more clarity, i'd say something like:

Note: on macOS, switching focus between windows does not trigger the `focus` and
`blur` events on `WebContents`. For a `WebContents` to have focus on macOS, it must be the [first responder](https://developer.apple.com/documentation/appkit/nswindow/1419440-firstresponder?language=objc) for its containing window, and switching focus between windows does not change the first responder.